app.json

```diff
@@ -34,6 +34,7 @@
     "splash": {
       "image": "./assets/splash.png",
       "backgroundColor": "#26C6A2"
-    }
+    },
+    "assets": ["./assets/fonts/"]
```

Comment:
You can remove this line since we already have installed the fonts through google fonts.